conans/client/cmd/export.py

```diff
@@ -81,12 +85,52 @@ def _load_export_conanfile(conanfile_path, output, name, version):
     return conanfile
 
 
+def _handle_cvs(conanfile_path, dest_folder, conanfile, output):
+    cvs_url = getattr(conanfile, "cvs_url", None)
+    if not cvs_url:
+        return
+
+    user_folder = os.path.dirname(conanfile_path).replace("\\", "/")
+    result_url = cvs_url
+    cvs_checkout = conanfile.cvs_checkout
+    result_checkout = cvs_checkout
+
+    modified = git_uncommitted(user_folder)
+
+    if modified:
+        result_checkout = "source"
+        result_url = user_folder
+    elif cvs_url == "auto":  # get origin
+        origin = git_origin(user_folder)
+        if origin:
+            result_url = origin
+        else:
+            result_url = user_folder
+
+    if not modified:
+        if cvs_checkout == "branch":
+            result_checkout = git_branch(user_folder)
+        elif cvs_checkout == "commit":
+            result_checkout = git_commit(user_folder)
+
+    final_path = os.path.join(dest_folder, "conanfile.py")
+    if result_checkout != cvs_checkout:
+        # FIXME: very fragile replace
+        replace_in_file(final_path, 'cvs_checkout = "%s"' % cvs_checkout,
+                        'cvs_checkout = "%s"' % result_checkout)
+
+    if result_url != cvs_url:
```

Comment:
Confusing replace of `cvs_url = "auto"`, the variable `cvs_url` could be different and we still need to replace with auto?